tests/xtimer_usleep_short/main.c

```diff
@@ -22,18 +22,54 @@
 #define TEST_USLEEP_MIN (0)
 #define TEST_USLEEP_MAX (500)
 
+#ifdef BOARD_NATIVE
+/* native can sometime take more time to respond as it is not real time */
+#define TEST_XTIMER_USLEEP_SHORT_SLEEP_MARGIN_US (1000)
+#else
+#define TEST_XTIMER_USLEEP_SHORT_SLEEP_MARGIN_US (20)
+#endif /* BOARD_NATIVE */
+
 int main(void)
 {
     xtimer_sleep(3);
```

Comment:
This `xtimer_sleep(3)` break my tests when doing `make flash test` on some boards as the test could match the `child.expect(u"This test will call xtimer_usleep for values from (\d+) down to (\d+)")` line before `make reset` is done.

Reply:
Any suggestions on how to fix this? Increase the sleep?